src/main/java/Task.java

```diff
@@ -0,0 +1,25 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : " "); //return tick symbol if task is done
```

Comment:
Perhaps this unicode can be replaced with a more expressive constant instead, so that the comment can be removed

Reply:
Got it! Fixed :)